### ISSUE TITLE
WA_TOKENKEY uses WA_MASTERKEY value

### ISSUE THREAD

**Issue by manojsitapara** [paulocorcino/baileys2api#2]
Thank you for converting Baileys library to API. I really appreciate your work. 

When I tried to use Environment file instead of config file, WA_TOKENKEY value is assigned from WA_MASTERKEY, I think it should be WA_TOKENKEY

[https://github.com/paulocorcino/baileys2api/blob/99d812da8ba04b59cb90f16c4924456d7108fae1/modules/whatsapp.js#L69](url)

**Comment by paulocorcino** [comment]
@manojsitapara Yes, this behavior exists due this was based to another API and I maintained compatible, but, my plan is near releases want remove MASTERKEY and continue only with TOKENKEY.
